src/background/modules/AutocorrectHandler.js

```diff
@@ -26,6 +26,76 @@ let antipatterns = [];
 // Adapted from: https://github.com/mozilla/webextension-polyfill/blob/master/src/browser-polyfill.js
 const IS_CHROME = Object.getPrototypeOf(browser) !== Object.prototype;
 
+/**
+ * Traverse Trie tree of objects to create RegEx.
+ *
+ * @param {Object.<string, Object|boolean>} obj
+ * @returns {string}
+ */
+function pattern(obj) {
+    const alt = [];
+    const cc = [];
+
+    // Escape special characters
+    const regExSpecialChars = /[.*+?^${}()|[\]\\]/gu;
+
+    for (const char in obj) {
+        if (char) {
+            const achar = char.replace(regExSpecialChars, "\\$&");
```

Comment:
How is `char` different frokm `achar`, the variabnle name does not make this obvious.

I think what you mean here is `escaptedChar`.

Reply:
Yes, `achar` is `char` with the special characters escaped. I will make this change.